apps/frontend/src/pages/quiz/libs/components/quiz-form/libs/helpers/extract-category-ids-from-questions.helper.ts

```diff
@@ -0,0 +1,18 @@
+import { ZERO_INDEX } from "~/libs/constants/constants.js";
```

Comment:
```suggestion
import { FIRST_ITEM_INDEX } from "~/libs/constants/constants.js";
```
Let's rename it or create a new one, ZERO_INDEX is not so clear